src/components/HTMLEngineProvider/HTMLRenderers/HereMentionRenderer.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import {TNodeChildrenRenderer} from 'react-native-render-html';
+import htmlRendererPropTypes from './htmlRendererPropTypes';
+import Text from '../../Text';
+import * as StyleUtils from '../../../styles/StyleUtils';
+
+const HereMentionRenderer = props => (
+    <Text>
+        <Text
+
+            // passing the true value to the function as here mention is always for the current user
+            color={StyleUtils.getUserMentionTextColor(true)}
+            style={StyleUtils.getUserMentionStyle(true)}
```

Comment:
I think we should probably change these to: 

`getMentionTextColor` and `getMentionStyle` since they aren't just for user mentions.